libdnf/base/solver_problems_impl.hpp

```diff
@@ -30,11 +32,12 @@ namespace libdnf::base {
 class SolverProblems::Impl {
 public:
     Impl() = default;
+    Impl(const SolverProblems::Impl & src) = default;
```

Comment:
This, OTOH, is not necessary, the compiler will generate the copy constructor if all the members are copyable.

Reply:
Fixed